Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -332,10 +333,20 @@ bool isThunderRanked()
 }
 #endif
 
+static void registerVideoEncoder()
```

Comment:
`registerWebRTCVideoEncoder` or explain better in the comment (as from what I see written I guess but don't dare extrapolating that the encoder can be also used for WebCodecs with a nice rename ?

Reply:
That element needs to be renamed and moved elsewhere in WebCore, it's on my TODO but there was quite a few changes in that code recently so i'm waiting a bit the dust to settle...

For now i'll go with `registerInternalVideoEncoder` I think.